samples/Localization.StarterWeb/wwwroot/lib/bootstrap/dist/js/bootstrap.min.js

```diff
@@ -0,0 +1,7 @@
+/*!
```

Comment:
In this case the project is exactly what a customer would get when do they File -> New in Visual Studio, which includes installing bower components to ~/wwwroot/lib. I admit it isn't ideal but it's the simplest thing and matches exactly with what customers get right now. Otherwise, I have to add an entire wwwroot/lib copy/management mechanism via gulp to this project and I really don't want to do that.